examples/kubernetes-config/kubernetes-clustersubnet.json

```diff
@@ -14,7 +14,7 @@
     },
     "agentPoolProfiles": [
       {
-        "name": "agent",
+        "name": "agentpool1",
```

Comment:
Changing this filename breaks our E2E tests, which consume them. Is there a good semantic reason to make a change? If so, we'll have to update our E2E test configurations.